documentation/docs/how_tos/mint_nft.mdx

```diff
@@ -0,0 +1,103 @@
+---
+description: "Learn how to mint NFTs with the iota.js client library."
+image: /img/client_banner.png
+keywords:
+- how to
+- example
+- nft
+- min
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import ExampleCode from '!!raw-loader!./../../../packages/iota/examples/mint-nft/src/index';
+
+# Mint an NFT
+
+## Pre-requirements
+
+Before you can mint an NFT, you will need to:
+
+* [Connect to a Node](simple.mdx).
+
+## Mint an NFT
+
+This [example](https://github.com/iotaledger/iota.js/tree/feat/stardust/packages/iota/examples/mint-nft) will connect to
+a node you defined as the `API_ENDPOINT` constant. Afterwards, the example will request a Bech32 address, if you don't
+have one the example will take care of generating it.
+
+### Create a Hot Wallet
+
+The example will later create a hot wallet and request funds from the
+[faucet](https://faucet.alphanet.iotaledger.net/api/enqueue) defined as the `FAUCET` constant.
+
+### Create the Output
+
+Once the generated wallet has outputs to mint the NFT, it will define and
+[INftOutput](../references/client/interfaces/INftOutput) object. This object.  The `amount` property is set to `0`, as
```

Comment:
Change 'and INftOutput'  to 'an INftOutput'  object. Remove leftover sentence "This object."